resolvers/projectResolver.ts

```diff
@@ -615,14 +615,18 @@ export class ProjectResolver {
       createdAt: new Date(),
       isMain: false
     })
-
+    const projectUpdateInfo = {
+      title: project.title,
+      email: project.users[0].email,
```

Comment:
I'll do a small change here @divine-comedian, project.users[0] will break for old projects.

Reply:
okay, what part needs to be changed?

Reply:
The user object is already the owner. So typing user.email should do it.

Reply:
so when I log user.email on  projectUpdate it returns undefined
however logging project.users[0].email returns the correct info